functions/Test-DbaValidLogin.ps1

```diff
@@ -281,42 +250,41 @@ Tests all logins excluding any that are from the subdomain Domain
 					continue
 				}
 				Write-Message -Message "Parsing Login $adlogin on $server" -Level Verbose
-				$exists = $enabled = $false
+				$exists = $false
 				if ($true)
 				{
 					$founduser = Get-DbaADObject -ADObject $adlogin -Type Group -Silent
 					if ($founduser) {
-						$exists = $enabled = $true
+						$exists = $true
 					}
 				}
 				else
 				{
 					Write-Warning -Message "AD Searcher Error for $groupname on $server" -Level Warning
 				}
-
-				if ($Detailed)
-				{
-					[PSCustomObject]@{
-						Server = $server.DomainInstanceName
-						Domain = $domain
-						Login = $groupname
-						Type = "Group"
-						Found = $exists
-						Enabled = $enabled
-						DisabledInSQLServer = $login.IsDisabled
-						ADLoginDetails = 'AD group'
-					}
+				$rtn = [PSCustomObject]@{
+					Server = $server.DomainInstanceName
+					Domain = $domain
+					Login = $groupname
+					Type = "Group"
```

Comment:
We'll better rename some properties to make clear we're talking about AD.
Maybe
[FoundInAD] or [ExistsInAD]?
[ADType]